app/views/offers/show.html.erb

```diff
@@ -1,32 +1,37 @@
 <div class="container">
-  <h1> <%= @offer.user.first_name%>'s Loan</h1>
-</div>
-<div class="offer-img">
-  <% if @offer.photo %>
-    <%= cl_image_tag @offer.photo, :style => "height: 30vh; width: 100vw; background-size: cover; object-fit: cover;"%>
-  <% else %>
-    <img src="https://images2.minutemediacdn.com/image/upload/c_fill,g_auto,h_1248,w_2220/f_auto,q_auto,w_1100/v1555347972/shape/mentalfloss/istock_000045501888_medium.jpg" alt="" class="user-img" style="height: 30vh; width: 100vw; background-size: cover; object-fit: cover;">
-  <% end %>
-</div>
-
-<div class="container" style="padding:30px 0 30px 0;">
+    <p><%= link_to "< Back to all loans", offers_path %></p>
+    <h1> <%= @offer.user.first_name%>'s Loan</h1>
   <div class="body">
-    <h3>Loan's conditions </h3>
-    <ul class="list-group list-group-horizontal">
-      <li class="list-group-item"><i class="fas fa-coins"></i> <%= @offer.amount ? @offer.amount : "none" %></li>
-      <li class="list-group-item"><i class="fas fa-euro-sign"></i> <%=@offer.currency ? @offer.currency : "none" %></li>
-      <li class="list-group-item"><i class="fas fa-percentage"></i> <%= @offer.interest ? @offer.interest : "none"  %></li>
-    </ul>
-  </div>
+    <div class="image">
+      <% if @offer.photo %>
+        <%= cl_image_tag @offer.photo, :style => "height: 50vh; width: 100%;"%>
+      <% else %>
+        <img src="https://images2.minutemediacdn.com/image/upload/c_fill,g_auto,h_1248,w_2220/f_auto,q_auto,w_1100/v1555347972/shape/mentalfloss/istock_000045501888_medium.jpg" alt="" class="user-img" style="height: 50vh; width: 100%;">
+      <% end %>
+    </div>
 
-  <div class="body-conditions">
-    <h3>about <%= @offer.user.first_name%>'s Loan</h3>
-    <p class="btn-flat">Request loan</p>
-  </div>
-  <div class="body-about">
-    <p><%= @offer.description ? @offer.description : "Lorem Ipsum es simplemente el texto de relleno de las imprentas y archivo" %></p>
-    <p class="btn-ghost">Contact lender</p>
-  </div>
+    <div class="content">
+      <div class="body-conditions">
+        <h3>Loan's conditions </h3>
+        <ul class="list-group list-group-horizontal">
+          <li class="list-group-item"><i class="fas fa-coins"></i> <%= @offer.amount ? @offer.amount : "none" %></li>
+          <li class="list-group-item"><i class="fas fa-euro-sign"></i> <%=@offer.currency ? @offer.currency : "none" %></li>
+          <li class="list-group-item"><i class="fas fa-percentage"></i> <%= @offer.interest ? @offer.interest : "none"  %></li>
```

Comment:
does it make sense to always have a EUR sign displayed, even when somebody would have doller as currency?
Right now we we always only have EUR, so only displaying the EUR sign is enough. Later we could do something like

case @offer.currency
when "EUR" then "€"
when "DOLLAR" then "$"

and so one



right now actually, amount, currency, and offer have to be set, because of: `validates presence: true`. So to clean it I would get rid of the if statement.